tests/components/nws/test_weather.py

```diff
@@ -188,7 +303,7 @@ async def test_error_observation(
 ) -> None:
     """Test error during update observation."""
     utc_time = dt_util.utcnow()
-    with patch("homeassistant.components.nws.utcnow") as mock_utc:
+    with patch("homeassistant.components.nws.coordinator.utcnow") as mock_utc:
```

Comment:
Ideally we use `FrozenDateTimeFactory` instead of patching `utcnow`

Reply:
I was trying to keep PR semi-manageable.  This was changed to accommodate the change in location, not a change in functionality.  I agree that these tests needs to be updated in the future.